test/python/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 function(manif_add_pytest target)
   add_test(
     NAME ${target}
-    COMMAND ${Python3_EXECUTABLE} -m pytest ${target}.py
+    COMMAND pytest-3 ${target}.py
```

Comment:
From where does this `pytest-3` come? I don't have it in my system and I never saw it before. I'm on ubuntu 20.04 with conda, but also in a fresh virtualenv I can't find it.

Reply:
Maybe calling `{python,python3} -m pytest` also here as done in CI?

Reply:
```
virtualenv /tmp/venv
source /tmp/venv/bin/activate
which pip # /tmp/bin/pip
pip install pytest
ls /tmp/venv/bin  # no pytest-3 here
```

Reply:
`pytest-3` comes along the `python3-pytest` `apt` packages.

The (totally arbitrary) line I've drawn here is that the plain CMake pipeline relies on `apt` packages. These targets are only really useful/consumed in conjunction with `make test`/`ctest`. Note how in the newly added `pybind11-cmake` CI job has no direct Python invoke (except at the very end to test the install/import).

Reply:
Often the test suite is executed as part of third-party packaging (e.g. conda / conda-forge) and it could be useful to make it work also outside the default CI upstream environment. However, from my point of view this is definitely not blocking and I'll proceed without it.

Reply:
Gotcha. I believe the existing conda-forge recipe uses the `pip install` path so this shouldn't be an issue atm.

Reply:
[Yes](https://github.com/conda-forge/manif-feedstock/blob/ecd36cbde64e2375f30d72654f88100680378c3c/recipe/build_py.sh#L3), and it just tests that the [import succeeds](https://github.com/conda-forge/manif-feedstock/blob/ecd36cbde64e2375f30d72654f88100680378c3c/recipe/meta.yaml#L60-L62).